[2024-05-09T02:14:07.534+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-09T02:14:07.592+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: my_dag_v1.training_model_B manual__2024-05-09T02:07:43.320809+00:00 [queued]>
[2024-05-09T02:14:07.613+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: my_dag_v1.training_model_B manual__2024-05-09T02:07:43.320809+00:00 [queued]>
[2024-05-09T02:14:07.614+0000] {taskinstance.py:2303} INFO - Starting attempt 3 of 8
[2024-05-09T02:14:07.658+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): training_model_B> on 2024-05-09 02:07:43.320809+00:00
[2024-05-09T02:14:07.673+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=3473) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-09T02:14:07.678+0000] {standard_task_runner.py:63} INFO - Started process 3478 to run task
[2024-05-09T02:14:07.681+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'my_dag_v1', 'training_model_B', 'manual__2024-05-09T02:07:43.320809+00:00', '--job-id', '44', '--raw', '--subdir', 'DAGS_FOLDER/my_dag.py', '--cfg-path', '/tmp/tmpsqyvx30i']
[2024-05-09T02:14:07.690+0000] {standard_task_runner.py:91} INFO - Job 44: Subtask training_model_B
[2024-05-09T02:14:07.845+0000] {task_command.py:426} INFO - Running <TaskInstance: my_dag_v1.training_model_B manual__2024-05-09T02:07:43.320809+00:00 [running]> on host 2b3626ba2b72
[2024-05-09T02:14:08.078+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='satyendra' AIRFLOW_CTX_DAG_ID='my_dag_v1' AIRFLOW_CTX_TASK_ID='training_model_B' AIRFLOW_CTX_EXECUTION_DATE='2024-05-09T02:07:43.320809+00:00' AIRFLOW_CTX_TRY_NUMBER='3' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-09T02:07:43.320809+00:00'
[2024-05-09T02:14:08.081+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-09T02:14:08.083+0000] {python.py:237} INFO - Done. Returned value was: 1
[2024-05-09T02:14:08.084+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-09T02:14:08.138+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=my_dag_v1, task_id=training_model_B, execution_date=20240509T020743, start_date=20240509T021407, end_date=20240509T021408
[2024-05-09T02:14:08.189+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-09T02:14:08.283+0000] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2024-05-09T02:14:08.295+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
